.changeset/honest-wasps-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+add `platform` to requests triggered by `fetch` in `load`
```

Comment:
Do we want to specify that this is just fixing this for requests that happen during SSR? (_Does_ it just fix it for requests that happen during SSR?) Not worth mentioning?

Reply:
It does only do this in SSR I believe, since it's about nested fetch requests in the same SSR event. Standard fetch requests already work.